notice_and_comment/static/regulations/css/less/fonts.less

```diff
@@ -0,0 +1,29 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.less
+ */
+
+@font-face {
```

Comment:
updated comment for clarity (two separate files). All font files would be a part of the same family and the different weights for each file are defined in the font-face declaration. Then the families would be declared the same for all headers and any that require specific treatment would just get a different weight property. 

This isn't a blocker though, I was just trying to understand a different way of doing the same thing. I  can approve/merge once backup fonts are added.